packages/client/src/Dashboard.tsx

```diff
@@ -239,7 +240,7 @@ function SelectedActionsBar({
       >
         <Toolbar className={classes.toolbar}>
           <span className={classes.title} />
-          <IconButton onClick={onArchive}>
+          <IconButton aria-label="archive" onClick={onArchive}>
```

Comment:
Adding an `aria-label` here is useful for accessibility, and it also gives me a selector I can use to find the archive button in tests.